packages/cactus-plugin-ledger-connector-xdai/src/test/typescript/integration/openapi/openapi-validation.test.ts

```diff
@@ -153,6 +152,7 @@ test(testCase, async (t: Test) => {
   test(`${testCase} - ${fDeploy} - ${cWithoutParams}`, async (t2: Test) => {
     try {
       const parameters = {
+        keychainId: "",
```

Comment:
@aldousalvarez Sorry, I made a mistake, this is a negative test case and the casting is necessary. You can get rid of the linter warning produced by the `any` if you instead cast to `unknown` first like you did in the beginning.

```suggestion
      const parameters = {
        contractName: HelloWorldContractJson.contractName,
        constructorArgs: [],
        web3SigningCredential: {
          ethAccount: whalePubKey,
          secret: whalePrivKey,
          type: Web3SigningCredentialType.PrivateKeyHex,
        },
        gas: 1000000,
      };
      await apiClient.deployContractV1(
        (parameters as unknown) as DeployContractV1Request,
      );
```